docs/reporters.md

```diff
@@ -39,19 +39,19 @@ inspec exec example_profile --reporter cli json:/tmp/output.json
 Output nothing to screen and write junit and html to a file.
 
 ```bash
-inspec exec example_profile --reporter junit:/tmp/junit.xml html2:www/index.html
+inspec exec example_profile --reporter junit2:/tmp/junit.xml html:www/index.html
```

Comment:
Is changing html2 to html a merge problem? I'd think we'd still want to guide people towards html2